src/lib/validations/sql.js

```diff
@@ -215,6 +215,15 @@ const checks: Checks = {
 		excerpt: 'Siteurl/home options',
 		recommendation: '',
 	},
+	siteHomeUrlLando: {
+		matcher: "'(siteurl|home)',\\s?'([^']+vipdev.lndo.site)'",
```

Comment:
```suggestion
		matcher: "'(siteurl|home)',\\s*'([^']+vipdev\\.lndo\\.site)'",
```

Reply:
@sjinks why the double escaping in `\\.`?

Reply:
@rinatkhaziev because this regexp is a string. We need smth like

```js
/'(siteurl|home)',\s*'([^']+vipdev\.lndo\.site)'/
```

But `"\."` is `"."`, and to get `\.`, we need to escape that slash as well.

Reply:
<img width="459" alt="Screenshot 2022-05-19 at 01 23 04" src="https://user-images.githubusercontent.com/7810770/169165455-78c660d6-28ba-4e8d-8114-c1284f760370.png">

Reply:
ah gotcha.